src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipsSelection.qml

```diff
@@ -37,15 +58,25 @@ Item {
 
         property real startX: 0
         property real startW: 0
+        property bool leftPressed: false
```

Comment:
May be name it "isDragging" to be more descriptive?

Reply:
named it that way because it's only true when left mouse button is held - onPositionChanged event does not recognize which button is held and we want to extend selection only when left mouse button is held

or isDragginEnabled maybe if you prefer?

Reply:
leftPressed is fine by me.  I believe that MouseArea.pressedButtons should contain the current mouse buttons pressed.

Reply:
yup, totally missed it, fixed